### PR TITLE
Updates to FADE batchfile

### PR DESCRIPTION
**Pull request by sjspielman**
Changes to FADE include the following:

1. Save equilibrium frequencies to JSON (also entailed a selection lib function update, and other batchfiles are updated to reflect this change)
2. Change order of estimation methods to have the default variational bayes first 
3. *Require* a rooted tree, which also makes prompts more consistent per inference.

Additional changes include ignoring a build directory and adding FADE to HyPhy prompt.

### CODE REVIEW

**Comment by sjspielman** [comment]
Second update to FADE simulator to save annotated tree to output so that labels are retained for downstream inference on simulated datasets.

**Comment by stevenweaver** [comment]
@sjspielman, 

Is this PR ready for review? Looking to release a new version of HyPhy.

Best,
Steven

**Comment by sjspielman** [comment]
Yes all ready, thanks!

File: res/TemplateBatchFiles/files.lst [hunk 1]
```diff
@@ -6,6 +6,7 @@
 "BUSTED","[BUSTED] Test for episodic gene-wide selection using BUSTED (Branch-site Unrestricted Statistical Test of Episodic Diversification).","SelectionAnalyses/BUSTED.bf";
 "aBSREL","[aBSREL] Test for lineage-specific evolution using the branch-site method aBS-REL (Adaptive Branch-Site Random Effects Likelihood).","SelectionAnalyses/aBSREL.bf";
 "RELAX","[RELAX] Test for relaxation of selection pressure along a specified set of test branches using RELAX (a random effects test of selection relaxation).","SelectionAnalyses/RELAX.bf";
+"FADE","[FADE] Test a protein alignment for directional selection towards specific amino acids along a specified set of test branches using FADE (a FUBAR Approach to Directional Evolution).","SelectionAnalyses/FADE.bf";
```
Comment (stevenweaver) [inline res/TemplateBatchFiles/files.lst]: Placing FADE in the menu prompt means that it is completely ready for release. I'm not sure if we should add it until the method is ready for publication.

**Pull request merged**